code/modules/projectiles/projectile_base.dm

```diff
@@ -212,7 +212,10 @@
 			return 0
 
 	var/distance = get_dist(get_turf(A), starting) // Get the distance between the turf shot from and the mob we hit and use that for the calculations.
-	def_zone = ran_zone(def_zone, max(100-(7*distance), 5)) //Lower accurancy/longer range tradeoff. 7 is a balanced number to use.
+	if(A == original)
+		def_zone = ran_zone(def_zone, max(100-(7*distance), 5)) //Lower accurancy/longer range tradeoff. 7 is a balanced number to use.
+	else
+		def_zone = pick(list("head", "chest", "l_arm", "r_arm", "l_leg", "r_leg")) // If we were aiming at one target but another one got hit, no accuracy is applied
```

Comment:
You may want to include the "groin" bodypart here, because it between the chest and legs. Also I don't know if you wanted to exclude the hands/feet, but I wanted to make sure

Reply:
no i dont want groin here
yes i wanted to remove feets and hands here